test/unit/search/query_components/booster_test.rb

```diff
@@ -2,54 +2,23 @@
     Timecop.freeze("2016-03-11 16:00".to_time) do
       builder = QueryComponents::Booster.new(search_query_params)
       result = builder.wrap({ some: 'query' })
 
-      expected = {
-        function_score: {
-          boost_mode: :multiply,
-          score_mode: "multiply",
-          query: {
-            bool: {
-              should: [
-                { some: "query" }
-              ]
-            }
-          },
-          functions: [
-            { filter: { term: { format: "service_manual_guide" } },    boost_factor: 0.3 },
-            { filter: { term: { format: "service_manual_topic" } },    boost_factor: 0.3 },
-            { filter: { term: { format: "smart-answer" } },            boost_factor: 1.5 },
-            { filter: { term: { format: "transaction" } },             boost_factor: 1.5 },
-            { filter: { term: { format: "topical_event" } },           boost_factor: 1.5 },
-            { filter: { term: { format: "minister" } },                boost_factor: 1.7 },
-            { filter: { term: { format: "organisation" } },            boost_factor: 2.5 },
-            { filter: { term: { format: "topic" } },                   boost_factor: 1.5 },
-            { filter: { term: { format: "document_series" } },         boost_factor: 1.3 },
-            { filter: { term: { format: "document_collection" } },     boost_factor: 1.3 },
-            { filter: { term: { format: "operational_field" } },       boost_factor: 1.5 },
-            { filter: { term: { format: "contact" } },                 boost_factor: 0.3 },
-            { filter: { term: { format: "aaib_report" } },             boost_factor: 0.2 },
-            { filter: { term: { format: "dfid_research_output" } },    boost_factor: 0.2 },
-            { filter: { term: { format: "hmrc_manual_section" } },     boost_factor: 0.2 },
-            { filter: { term: { format: "service_standard_report" } }, boost_factor: 0.2 },
-            { filter: { term: { format: "mainstream_browse_page" } },  boost_factor: 0 },
-            { filter: { term: { search_format_types: "announcement" } },
-              script_score: {
-                script: "((0.05 / ((3.16*pow(10,-11)) * abs(now - doc['public_timestamp'].date.getMillis()) + 0.05)) + 0.12)",
-                params: { now: 1457712000000 }
-              }
-            },
-            { filter: { term: { organisation_state: "closed" } }, boost_factor: 0.2 },
-            { filter: { term: { organisation_state: "devolved" } }, boost_factor: 0.3 },
-            { filter: { term: { is_historic: true } }, boost_factor: 0.5 }
-          ]
-        }
-      }
+      assert_equal :multiply, result[:function_score][:boost_mode]
+      assert_equal :multiply, result[:function_score][:score_mode]
 
-      assert_equal expected, result
+      assert_boost(result, "organisation", 1.0)
+      assert_boost(result, "service_manual_guide", 0.3)
+      assert_boost(result, "mainstream_browse_page", 0)
```

Comment:
I didn't test every format because we're basically testing config. My aim was to add a few regression tests for the logic without making the tests fragile. (The original tests tripped me up because they were so tied to the exact output of `builder.wrap`.) What do you think?

Good spot on the non-format cases. I've added some more tests to cover those.